geth/api/backend.go

```diff
@@ -239,7 +239,7 @@ func (b *StatusBackend) ConnectionChange(state ConnectionState) {
 
 // AppStateChange handles app state changes (background/foreground).
 func (b *StatusBackend) AppStateChange(state AppState) {
-	log.Info("App State changed: %s", state)
+	log.Info(fmt.Sprintf("App State changed: %s", state))
```

Comment:
"app changed" "state" state ?

Reply:
Yes, like with errors the logging is nicer when printing a message and then key/value pairs.